client/app/widgets/ToggleLocale/__tests__/ToggleLocale.spec.jsx

```diff
@@ -17,23 +22,25 @@ describe('ToggleLocale', () => {
   });
 
   it('does nothing if the previous locale is the same as the selected', () => {
-    const wrapper = mount(component);
+    render(component);
     act(() => {
-      wrapper.find('select').prop('onChange')({
-        currentTarget: { value: 'en' },
+      fireEvent.select(screen.getByRole('combobox'), {
```

Comment:
The dev dependency '@testing-library/user-event' is also available for simulating user behavior:

```
import userEvent from '@testing-library/user-event';

...

userEvent.selectOptions(screen.getByRole('combobox'), 'en');
```

It has some useful abstractions outside of `fireEvent`, as seen with the `selectOptions` [method](https://github.com/testing-library/user-event#selectoptionselement-values) used in this example.

Reply:
I didn't notice that we are already using it! Thank you for pointing that out.